stream-chat-android-compose-sample/src/main/java/io/getstream/chat/android/compose/sample/ui/MessagesActivity.kt

```diff
@@ -298,10 +300,19 @@ class MessagesActivity : BaseConnectedActivity() {
 
     companion object {
         private const val KEY_CHANNEL_ID = "channelId"
+        private const val KEY_MESSAGE_ID = "messageId"
+        private const val KEY_PARENT_MESSAGE_ID = "parentMessageId"
 
-        fun createIntent(context: Context, channelId: String): Intent {
+        fun createIntent(
+            context: Context,
+            channelId: String,
+            messageId: String? = null,
+            parentMessageId: String?,
```

Comment:
Can we add a default param to this argument?
It won't be null only on the case we need to navigate to a thread message (What should be really few times)